src/app/screens/Enable/index.tsx

```diff
@@ -30,6 +31,19 @@ function Enable(props: Props) {
     event.preventDefault();
   }
 
+  async function block(event: React.MouseEvent<HTMLAnchorElement>) {
+    msg.error(
+      `User added site to blocklist domain, host
+        ${props.origin.domain},
+        ${props.origin.host}`
+    );
+    await utils.call("addBlocklist", {
```

Comment:
@escapedcat at some point I wanted to namespace these calls. 
e.g. `utils.call("blocklist/add")` to have this grouped in the background-script. 
I did not use it for simplicity. but now that we get more and more calls we maybe slowly could think about this again.

Reply:
That's a good idea! Created: https://github.com/getAlby/lightning-browser-extension/issues/1065